tests/test_elb.py

```diff
@@ -614,3 +614,204 @@ def test_is_logging_to_bucket(self):
         self.assertGreater(
             len(resources), 0, "Should find elb not logging " "to otherbucket"
         )
+
+
+class TestElbAttributeFilter(BaseTest):
+
+    def test_is_connection_draining(self):
+        """ replicate
+            - name: elb-is-connection-draining-test
+              resource: elb
+              filters:
+                - type: attributes
+                  key: ConnectionDraining.Enabled
+                  value: true
+                  op: eq
+        """
+        session_factory = self.replay_flight_data("test_elb_attribute_filter")
+        policy = self.load_policy(
+            {
+                "name": "elb-is-connection-draining-test",
+                "resource": "elb",
+                "filters": [
+                        {
+                            "type": "attributes",
+                            "key": "ConnectionDraining.Enabled",
+                            "value": True,
+                            "op": "eq"
+                        }
+                ],
+            },
+            session_factory=session_factory,
+        )
+
+        resources = policy.run()
+
+        self.assertEqual(
+            len(resources), 1, "Test should find one elb connection draining"
+        )
```

Comment:
It would be good to add a secondary assertion to assert the thing we filtered on, for example:

```
       self.assertEqual(resources[0][Attributes][ConnectionDraining][Enabled], True)
```